cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -291,10 +295,42 @@ def _clear_ftp():
         ftp.recursively_delete()
 
     # Start the collection
+    collect_time = datetime.utcnow() - timedelta(hours=4)  # time on dropbox is UTC-4
     configured_depot.collect_all()
     # Check it on FTP
-    check_ftp(ftp=log_depot.ftp, server_name=appliance.server.name,
-              server_zone_id=appliance.server.zone.id, check_ansible_logs=True)
+    if log_depot.protocol != 'dropbox':
+        check_ftp(ftp=log_depot.ftp, server_name=appliance.server.name,
+                  server_zone_id=appliance.server.zone.id, check_ansible_logs=True)
+    elif appliance.is_downstream:  # check for logs on dropbox, not applicable for upstream
+        username = conf.credentials['rh_dropbox']['username']
```

Comment:
Please wrap this to catch the `KeyError` when the credentials don't include rh_dropbox and skip the test